src/style.css

```diff
@@ -0,0 +1,119 @@
+:root {
+  --text-dark: #3d4e61;
+  --border-gray: #e1e7ed;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: 900;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.bigForm {
```

Comment:
I would recommend to use not camelCase for BEM naming. You have already applied BEM-modifier for icons. 
So use `form`, `input`, `icon`, `label` styles for the FIRST form where you apply generic styles for both forms
And use `form form--small`, `input input--small`, `icon icon--small`, `label -label--small` for the SECOND form
So you will follow BEM principe